index.html

```diff
@@ -3191,12 +3191,13 @@ <h2>Integrity of Related Resources</h2>
         </p>
 
         <p>
-Specification authors that write algorithms that fetch a resource based on the
-`id` of an object inside a [=conforming document=] need to consider whether
-that resource's content is vital to the validity of that document. If it is, the
-specification MUST produce a validation error unless the resource matches the
-expected media type and cryptographic digest.
+A [=conforming verifier implementation=] that makes use of a resource based on
+the `id` of an object inside a [=conforming document=] with a corresponding
```

Comment:
If necessary for clarity, here's an alternative suggestion to what Ivan suggests below:

```suggestion
the `id` of a `relatedResource` object inside a [=conforming document=] with a corresponding
```

Reply:
Yep, that works and is mathematically more precise.

👍